### ISSUE TITLE
Bug: Cannot get column for filter

### ISSUE THREAD

**Issue by hongquan** [mrjoes/flask-admin#668]
When I define `column_filters` with `BaseFilter` objects, like this

```
column_filters = [FilterLike(BusinessOwner.gateway_name, 'Gateway')]
```

I get this error:

```
Exception: Invalid field <flask_admin.contrib.sqla.filters.FilterLike object at 0x7f7e3bb9e860>: does not contains any columns.
```

It is raised from here:
https://github.com/mrjoes/flask-admin/blob/v1.0.8/flask_admin/contrib/sqla/view.py#L390

Looking into the parent classes (`BaseFilter`, `BaseSQLAFilter`), I found no place where `property` attribute is defined for these classes.

This issue happens with Python 3.4.2, not Python 3.4.1.

**Comment by torotil** [comment]
I've got the same error and the same workaround solves it too. I think importing via flask_admin should be supported because AFAIK that's the way flask is going right now.

**Comment by hongquan** [comment]
I think this issue is fixed in 222261a.
